$OpenBSD: patch-Modules_FindGettext_cmake,v 1.2 2012/11/26 12:32:29 dcoppa Exp $

Fix overwriting result with empty variable
(upstream git commit 94f37851ef5c6dccbe2dd647a3b38e3196393298)

--- Modules/FindGettext.cmake.orig	Mon Nov 26 12:30:02 2012
+++ Modules/FindGettext.cmake	Mon Nov 26 12:30:34 2012
@@ -209,5 +209,3 @@ function(GETTEXT_PROCESS_PO_FILES _lang)
    add_dependencies(pofiles ${uniqueTargetName})
 
 endfunction()
-
-set(GETTEXT_FOUND ${Gettext_FOUND})
